dsst_etl/upload_pdfs_title_is_pmid.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+import hashlib
+
+import boto3
+import requests
+import sqlalchemy
+
+from dsst_etl import __version__, logger
+from dsst_etl._utils import get_bucket_name, get_compute_context_id
+from dsst_etl.models import Documents, Identifier, OddpubMetrics, Provenance, Works
+
+from .config import config
+
+
+class UploadPDFsTitleIsPMID:
+    """
+    Uploads PDFs to S3 where the title is the PMID.
+    """
+
+    def __init__(
+        self,
+        db_session: sqlalchemy.orm.Session,
+        oddpub_host_api: str = config.ODDPUB_HOST_API,
+    ):
+        self.db_session = db_session
+        self.bucket_name = get_bucket_name()
+        self.s3_client = boto3.client("s3")
+        self.oddpub_host_api = oddpub_host_api
+
+    def process_s3_bucket(self) -> str:
```

Comment:
I think this method could be named better. Adding a docstring might facilitate the naming. Perhaps just `run`?

Reply:
You seem to be doing two things here, iterating over a bucket's contents for inventory and running oddpub. You could name it
```suggestion
    def pmid_inventory_oddpub(self) -> str:
```